app/views/procedures/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<div>
+  <%=@procedure.name%>
+</div>
+
+<div>
+  <%=f.description%>
```

Comment:
What is `f.description` here? It doesn't look like this is part of a form, which is when `f.` is typically used. Is this supposed to be `@procedure.description` perhaps?

Reply:
whoa, yes good catch.